src/Insurance/index.ts

```diff
@@ -0,0 +1,36 @@
+import BigNumber from "bignumber.js";
+
+const HOURLY_COMPOUND_FREQUENCY = new BigNumber(24*365); // hourly insurance funding rate
```

Comment:
Are we using hourly or eight hourly, I thought we were using 8 hourly but webby said hourly

Reply:
@dospore hourly atm